workflows/nanostring.nf

```diff
@@ -73,23 +74,24 @@ workflow NANOSTRING {
     //
     // SUBWORKFLOW: Quality control of input files
     //
-    QUALITY_CONTROL (
+    NACHO_QC (
         rcc_files,
         samplesheet_path.first()
     )
-    ch_versions      = ch_versions.mix(QUALITY_CONTROL.out.versions)
-    ch_multiqc_files = ch_multiqc_files.mix(QUALITY_CONTROL.out.nacho_qc_multiqc_metrics.collect())
+    ch_versions      = ch_versions.mix(NACHO_QC.out.versions)
+    nacho_qc_multiqc_metrics = NACHO_QC.out.nacho_qc_png.map{it[1]}.mix(NACHO_QC.out.nacho_qc_txt.map{it[1]})
```

Comment:
By convention (at least in this pipeline) we prefix FIFO channel variables with `ch_`. 

```suggestion
    ch_nacho_qc_multiqc_metrics = NACHO_QC.out.nacho_qc_png.map{it[1]}.mix(NACHO_QC.out.nacho_qc_txt.map{it[1]})
```